workflows/phyloplace.nf

```diff
@@ -15,35 +15,38 @@ for (param in checkPathParamList) { if (param) { file(param, checkIfExists: true
 
 // Check mandatory parameters
 if (params.input) { 
-    //ch_input = Channel.fromPath(params.input) 
     Channel.fromPath(params.input)
         .splitCsv(header: true)
         .map { 
             [ 
                 meta: [ id: it.sample ], 
                 data: [ 
```

Comment:
Just out of curiousity: why are all the files also in a separate list (like meta)?

Reply:
I actually don't remember. Either I didn't get it to work without a second list, or I followed an example :-S.

Reply:
Oooh I see, well as long as it works :)